### PR TITLE
Created the getSeed()

### PR DESCRIPTION
**Pull request by cauelorenzato**
To use phinx for unit testing, sometimes is necessary to run not just the migrations, but the seeds as well. This commit add a new method getSeed that runs the seeds.

### CODE REVIEW

**Comment by rquadling** [comment]
Looking to get this bundled into a release for the end of the week. I may come back with further questions, but OK so far.

**Comment by rquadling** [comment]
Thank you.

File: src/Phinx/Wrapper/TextWrapper.php [hunk 1]
```diff
@@ -125,6 +125,26 @@ public function getMigrate($env = null, $target = null)
     }
 
     /**
+     * Returns the output from running the "seed:run" command.
+     * @param  string $env environment name (optional)
+     * @param  string $target target version (optional)
+     * @return string
+     */
+    public function getSeed($env = null, $target = null)
```
Comment (flavioheleno) [inline src/Phinx/Wrapper/TextWrapper.php]: If I can suggest, add a third parameter `$seed = null`, which would be the seed's name and is optional.

File: src/Phinx/Wrapper/TextWrapper.php [hunk 2]
```diff
@@ -125,6 +125,26 @@ public function getMigrate($env = null, $target = null)
...
+    {
+        $command = array (
+            'seed:run',
+            '-e' => $env?: $this->getOption('environment'),
+            '-c' => $this->getOption('configuration'),
+            '-p' => $this->getOption('parser')
+        );
+        if ($target) {
+            $command += array('-t' => $target);
+        }
```
Comment (flavioheleno) [inline src/Phinx/Wrapper/TextWrapper.php]: Then add support to the `$seed` parameter as:
```php
if ($seed) {
    $command += array('-s' => $seed);
}
```

**Comment by flavioheleno** [comment]
hey @cauelorenzato and @rquadling I've added a few comments that would improve `getSeed` usage in cases where you don't want to run all seeds but just a single seed.

**Comment by rquadling** [comment]
Excellent. Can you squash the commits please?

**Comment by rquadling** [comment]
In future, you do not need to create a new PR. You can do a force push. Especially useful after a rebase when you are working on your PRs for someone else's code.

Replaced by https://github.com/robmorgan/phinx/pull/921

**Pull request closed without merging**